src/Agent.Sdk/Util/PlatformUtil.cs

```diff
@@ -315,8 +315,9 @@ private async static Task<OperatingSystem[]> GetNet6SupportedSystems()
             string supportOSfilePath = Path.Combine(Path.GetDirectoryName(Assembly.GetEntryAssembly().Location), "net6.json");
             string supportOSfileContent;
 
-            if (!File.Exists(supportOSfilePath) || File.GetLastWriteTimeUtc(supportOSfilePath) < DateTime.UtcNow.AddHours(-1))
+            if (!File.Exists(supportOSfilePath) || File.GetLastWriteTimeUtc(supportOSfilePath) < DateTime.UtcNow.AddDays(3))
```

Comment:
In PR description you mentioned 7 days. What time should be in result?

Reply:
Updated PR's description, but thanks for mentioned. Because I need to extract date but not add. Fixed